app/services/yelp_service.rb

```diff
@@ -0,0 +1,25 @@
+class YelpService
+
+  attr_reader :client
+
+  def city_search(city)
+    client.search(city)
+  end
+
+  def box_search(box, params)
+    client.search_by_bounding_box(box, params)
+  end
+
+  def radius_search(coordinates, search_terms, radius = 1000, sort = 0)
+    coordinates_hash = { latitude: coordinates[0], longitude: coordinates[1] }
+    client.search_by_coordinates(coordinates_hash, term: search_terms, radius_filter: radius, sort: sort)
+  end
+
+  def client
+    @client = Yelp::Client.new({ consumer_key: ENV['yelp_consumer_key'],
```

Comment:
Redundant curly braces around a hash parameter.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.